app/api/V2/__init__.py

```diff
@@ -32,3 +33,4 @@
 api.add_resource(UserRegistration, '/auth/signup')
 api.add_resource(UserLogin, '/auth/login')
 api.add_resource(TokenRefresh, '/auth')
+api.add_resource(GetAllUsers, '/users')
```

Comment:
Leave one line at the end of each file. Why I'm I repeating the same feedback everytime I'm reviewing your PRs?